README.rst

```diff
@@ -33,7 +33,31 @@ You can also set the `TESTING` flag to `True` so no mails are sent
 Deploy it
 =========
 
-To deploy it, I'm using gunicorn and supervisord.
+With Apache and mod_wsgi
+------------------------
+
+1. Install Apache and mod_wsgi::
+
+    $ sudo apt-get install apache2
```

Comment:
It seems that on new debian versions you can use `apt install` directly, should we propose that instead?